docs/reference/primitives/app/resource/memcached.md

```diff
@@ -8,23 +8,92 @@ url: /reference/primitives/app/resource/memcached
-```html
+
+### AWS ElastiCache Managed Memcached Resources
+
+In addition to containerized resources, Convox v3 allows the creation of Memcached resources via AWS ElastiCache. This provides enhanced durability and managed service benefits. Below is a general example of how to define AWS ElastiCache Memcached resources:
+
+```
+resources:
+  cache:
+    type: elasticache-memcached
+    options:
+      class: cache.t3.micro
+      version: 1.6.6
+      deletionProtection: true
+      durable: true
+      encrypted: true
+      autoMinorVersionUpgrade: true
+      nodes: 2
+services:
+  web:
     resources:
-      main:
-        type: memcached
-        options:
-          version: 1.4.34
-```
\ No newline at end of file
+      - cache
+```
+
+> **Note:** The `nodes` option is required for Memcached and must be set to the desired number of nodes for the Memcached cluster.
+
+### Features
+
+- **Scalable Memcached Instances**: AWS ElastiCache Memcached supports multiple nodes for improved scalability, making it an ideal caching solution for distributed applications.
+- **Import Existing ElastiCache Memcached Instance**: You can import an existing AWS ElastiCache Memcached instance into a Convox rack for management or access via linking.
+
+### Configuration Options
+
+Below is a chart of configuration options available for AWS ElastiCache Memcached resources:
+
+| Attribute                | Type    | Description                                                                                       |
+| ------------------------ | ------- | ------------------------------------------------------------------------------------------------- |
+| **nodes**                | int     | **Required.** The number of nodes in the Memcached cluster.                                        |
+| **class**                | string  | The compute and memory capacity of the cache instance.                                             |
+| **version**              | string  | The version of the cache engine.                                                                   |
+| **deletionProtection**   | boolean | Whether to enable deletion protection for the cache instance.                                      |
+| **durable**              | boolean | Whether to create a Multi-AZ cache instance.                                                       |
+| **encrypted**            | boolean | Whether to enable encryption at rest for the cache instance.                                       |
+| **autoMinorVersionUpgrade** | boolean | Whether to allow automatic minor version upgrades for the cache engine.                           |
+| **import**                    | string  | The cache identifier used for cache import. Requires the correct `masterUserPassword` option set.       |
+| **masterUserPassword**        | string  | The password for the master user. Should be set as an environment variable to avoid hardcoding.               |
```

Comment:
can you remove masterUserPassword?
cache doesn't have that

Reply:
fixed